_posts/2020-08-07-g2p.md

```diff
@@ -94,23 +96,25 @@ or like this using JSON:
 ]
 ```
 
-## Ok, so how do I actually get started here?
+## Ok, so how do I actually get these rules to do something?
```

Comment:
After explaining the "why" in the previous text, I would make this section the start of a new post: "Creating your first g2p mapping". I'd like this to be a beginner-friendly tutorial, essentially, as its own standalone blog post!

Overall in this section, I'd like more examples of concrete rules, and how do I come up with a rule in the first place? How do I know when to make a rule?